plugin/storage/badger/spanstore/reader.go

```diff
@@ -172,6 +172,10 @@ func (r *TraceReader) GetTrace(ctx context.Context, traceID model.TraceID) (*mod
 	return nil, ErrInternalConsistencyError
 }
 
+func (r *TraceReader) GetTraceForOrg(ctx context.Context, traceID model.TraceID, orgId model.OrgId) (*model.Trace, error) {
+	return nil, nil
```

Comment:
should this return null?

Reply:
oh, is this because it's not implemented? maybe add a comment?

Reply:
will do.